README.md

```diff
@@ -1,60 +1,189 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
-# Portfolio Project
+<div align="center">
 
-> My Complete Portfolio 
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
+  <h3><b>Microverse README Template</b></h3>
 
-## Built With
+</div>
 
-- Html
-- CSS
+<!-- TABLE OF CONTENTS -->
 
-## Live Demo 
+# 📗 Table of Contents
 
-[Live Demo Link](https://rana-mudassir.github.io/portfolio_project/)
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
```

Comment:
[REQUIRED] Good job using the new template but you should also fill it up with your details